src/components/Forms/ComboBox.jsx

```diff
@@ -0,0 +1,94 @@
+              className="w-full border-none focus:ring-0 py-2 pl-3 pr-10 text-sm leading-5 text-primary bg-background rounded-lg"
+              displayValue={(option) => option}
+              onChange={(event) => setQuery(event.target.value)}
+            />
+            <Combobox.Button className="absolute inset-y-0 right-0 flex items-center pr-2">
+              <HiSelector
+                className="w-5 h-5 text-secondary"
+                aria-hidden="true"
+              />
+            </Combobox.Button>
+          </div>
+          <Transition
+            as={Fragment}
+            leave="transition ease-in duration-100"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
+            afterLeave={() => setQuery('')}
+          >
+            <Combobox.Options className="absolute w-80 py-1 mt-1 overflow-auto text-base bg-background text-primary font-medium rounded-md shadow-lg max-h-40 ring-1 ring-primary ring-opacity-5 focus:outline-none sm:text-sm h-52 z-50 left-4 border border-secondary">
+              {filteredOptions.length === 0 && query !== '' ? (
+                <div className="cursor-default select-none relative py-2 px-4 text-primary">
+                  Nothing found.
+                </div>
+              ) : (
+                filteredOptions.map((option) => (
+                  <Combobox.Option
+                    key={option.id || option}
+                    className={({ active }) =>
+                      `cursor-default select-none relative py-2 pl-10 pr-4 text-left ${
+                        active
+                          ? 'text-white bg-secondary bg-opacity-50'
+                          : 'text-primary'
+                      }`
+                    }
+                    value={option}
+                  >
+                    {({ selected, active }) => (
+                      <>
+                        <span
+                          className={`block truncate ${
+                            selected ? 'font-medium' : 'font-normal'
+                          }`}
+                        >
+                          {option.name || option}
+                        </span>
+                        {selected ? (
+                          <span
+                            className={`absolute inset-y-0 left-0 flex items-center pl-3 ${
+                              active ? 'text-secondary' : 'text-tertiary'
+                            }`}
+                          >
+                            <HiOutlineCheck
+                              className="w-5 h-5"
+                              aria-hidden="true"
+                            />
+                          </span>
+                        ) : null}
+                      </>
+                    )}
+                  </Combobox.Option>
+                ))
+              )}
+            </Combobox.Options>
+          </Transition>
+        </div>
+      </Combobox>
+    </div>
+  );
+}
+
+export default ComboBox;
```

Comment:
Nice usage of combobox here, but it does not allow us to select multiple item types which is a needed functionality, can we change this so that it lets us hold multiple types ?